src/signTx.c

```diff
@@ -1457,12 +1489,9 @@ static void signTx_handleWithdrawalAPDU(uint8_t p2, uint8_t* wireDataBuffer, siz
 		read_view_t view = make_read_view(wireDataBuffer, wireDataBuffer + wireDataSize);
 		VALIDATE(view_remainingSize(&view) >= 8, ERR_INVALID_DATA);
 		txBodyCtx->stageData.withdrawal.amount = parse_u8be(&view);
-		// the rest is path
+		// the rest is the identifier, either path or scripthash
```

Comment:
identifier -> stake credential?
Or I would remove the comment, the next line is self-explanatory.

Reply:
Removed.